### ISSUE TITLE
Ford should have a better icon

### ISSUE THREAD

**Issue by cicku** [openstreetmap/iD#8691]
Currently even on the motorcycle road it still displays as a pedestrian crossing, which does not look well.

![image](https://user-images.githubusercontent.com/686438/133268023-6df4aca1-ef07-4ae9-93fd-e9aca2aec57f.png)

**Comment by bhousel** [comment]
I agree - we have an issue here for it: https://github.com/ideditor/temaki/issues/48

**Comment by cicku** [comment]
Wow nice. I could send a design there when I have time. When it is a motorcycle ford, it could be:

```
car icon
_______
~~~~~~
```

The above is the prototype in my opinion.

**Comment by boothym** [comment]
What about something like osm-carto's double arrow + wavy line? (at least the bottom bit anyway)

https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/highway/ford.svg

**Comment by k-yle** [comment]
this was fixed in https://github.com/openstreetmap/id-tagging-schema/commit/c0c94c0a6ddf58cd92a95cfa399bade027b28690
